usr/share/rear/verify/BAREOS/default/100_bextract_mount.sh

```diff
@@ -10,10 +10,4 @@ if [ "$BEXTRACT_DEVICE" -o "$BEXTRACT_VOLUME" ]; then
       StopIfError "Could not mount Bareos device $BAREOS_DEVICE at /backup"
```

Comment:
Regardless that this line is not changed by this pull request:
In general please avoid the ...IfError functions,
for the reson behind see my other comment above.